tensorboard/plugins/interactive_inference/witwidget/notebook/colab/wit.py

```diff
@@ -233,9 +232,13 @@ def set_examples(self, examples):
       self._generate_sprite()
 
   def infer(self):
-    inferences = base.WitWidgetBase.infer_impl(self)
-    output.eval_js("""inferenceCallback('{inferences}')""".format(
-      inferences=json.dumps(inferences)))
+    try:
+      inferences = base.WitWidgetBase.infer_impl(self)
+      output.eval_js("""inferenceCallback({inferences})""".format(
+        inferences=json.dumps(inferences)))
```

Comment:
as per http://google.github.io/styleguide/pyguide.html#3192-line-breaking, you might want +2 more whitespace here and below.

Reply:
done